src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -1040,6 +1040,55 @@ public enum KeyContainerPermissionFlags
         ChangeAcl = 8192,
         AllFlags = 13111,
     }
+    public sealed partial class MediaPermission : System.Security.CodeAccessPermission, System.Security.Permissions.IUnrestrictedPermission
+    {
+        public MediaPermission() { }
+        public MediaPermission(System.Security.Permissions.MediaPermissionAudio permissionAudio) { }
+        public MediaPermission(System.Security.Permissions.MediaPermissionAudio permissionAudio, System.Security.Permissions.MediaPermissionVideo permissionVideo, System.Security.Permissions.MediaPermissionImage permissionImage) { }
+        public MediaPermission(System.Security.Permissions.MediaPermissionImage permissionImage) { }
+        public MediaPermission(System.Security.Permissions.MediaPermissionVideo permissionVideo) { }
+        public MediaPermission(System.Security.Permissions.PermissionState state) { }
+        public System.Security.Permissions.MediaPermissionAudio Audio { get { throw null; } }
+        public System.Security.Permissions.MediaPermissionImage Image { get { throw null; } }
+        public System.Security.Permissions.MediaPermissionVideo Video { get { throw null; } }
+        public override System.Security.IPermission Copy() { throw null; }
+        public override void FromXml(System.Security.SecurityElement securityElement) { }
+        public override System.Security.IPermission Intersect(System.Security.IPermission target) { throw null; }
```

Comment:
For overridden methods, perhaps forward the calls to `base` as a general practice? @rladuca thoughts?

Reply:
So looking at `CodeAccessPermission`, some of these things marked override are abstract.  I think we should categorize the overrides and if any of them provide more utility (from some other base class maybe) than throw null, then call them.  But at a glance, I don't see anything that useful.